src/main/java/com/aspectsecurity/automation/testing/JavaParser/visitors/SpringAnnotationAnalyzer.java

```diff
@@ -129,67 +124,59 @@ else if (child.getClass().equals("com.github.javaparser.ast.expr.NameExpr"))
         return parameters;
     }
 
-    private String getClassNameFromMethod(MethodDeclaration n)
-    {
+    private String getClassNameFromMethod(MethodDeclaration n) {
         Optional<ClassOrInterfaceDeclaration> methodClazzNode;
-        String methodClazzName;
+        String methodClazzName = null;
```

Comment:
Same here - we should do a quick review to ensure a null value doesn't throw an unhandled exception.

Reply:
So upon review, this should never be an issues since methods should always be part of a class. A NPE here would mean someone is doing something very wrong... I'm ok with your update.